e2e/jest.test-setup.ts

```diff
@@ -14,9 +12,9 @@ const isDebugMode = process.argv.includes('--debug');
  */
 beforeEach(async () => {
   await page.setUserAgent(userAgent);
-  // See https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetdefaultnavigationtimeouttimeout
-  page.setDefaultNavigationTimeout(navTimeOut);
-  page.setDefaultTimeout(timeOut);
+  await page.setViewport({width: 1280, height: 680});
```

Comment:
`width:1280, height:680` is based on 15" mac pro screen size.

Reply:
My [first computer](https://en.wikipedia.org/wiki/Macintosh_Plus#:~:text=A%20compact%20Mac%2C%20the%20Plus,that%20of%20previous%20Macintosh%20models.) only had 512x342. 👴

Reply:
right. 1280 x 680 actually is the minimum size where workbench UI does not have a horizontal scrollbar.